docker-compose.yml

```
@@ -0,0 +1,34 @@
+      - ./data:/app/data
+    # depends_on:  # Descomenta si usas PostgreSQL
+    #   - postgres
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8000/"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)